common/containers/Tabs/WalletDecrypt/index.jsx

```diff
@@ -0,0 +1,117 @@
+                                                aria-labelledby="mnemonic-phrase-label"
+                                                type="radio"
+                                                value="pastemnemonic"
+                                                name="137"
+                                            />
+                                            <span id="mnemonic-phrase-label">
+                                                {translate('x_Mnemonic')}
+                                            </span>
+                                        </label>
+                                        <label className="radio">
+                                            <input
+                                                aria-flowto="aria4"
+                                                aria-labelledby="parity-phrase-label"
+                                                type="radio"
+                                                value="parityBWallet"
+                                                name="139"
+                                            />
+                                            <span id="parity-phrase-label">
+                                                {translate('x_ParityPhrase')}
+                                            </span>
+                                        </label>
+                                        <label className="radio">
+                                            <input
+                                                aria-flowto="aria5"
+                                                aria-labelledby="ledger-nano-s-label"
+                                                type="radio"
+                                                value="ledger"
+                                                name="141"
+                                            />
+                                            <span id="ledger-nano-s-label">
+                                                {translate('x_Ledger')}
+                                            </span>
+                                        </label>
+                                        <label className="radio">
+                                            <input
+                                                aria-flowto="aria6"
+                                                aria-labelledby="trezor-label"
+                                                type="radio"
+                                                value="trezor"
+                                                name="142"
+                                            />
+                                            <span id="trezor-label">
+                                                {translate('x_Trezor')}
+                                            </span>
+                                        </label>
+                                        <label className="radio">
```

Comment:
@dternyak You're absolutely right. I think I will break these out and do that.

I was so fixated on the markup that I forgot to write it right 😢